packages/core/contracts/bot-helpers/atomic-liquidator/ReserveCurrencyLiquidator.sol

```diff
@@ -2,6 +2,7 @@ pragma solidity ^0.6.0;
 pragma experimental ABIEncoderV2;
```

Comment:
This contract was basically re-written with a bunch more logic within it. In particular, the reserve currency liquidator now considers any existing synthetic or colalteral tokens the DSproxy might have. For example, if we manually sent the DSProxy a bunch of synthetic tokens it should use these before minting extras.